index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<main>
+    <h1 id="title"> Survey Form </h1>
```

Comment:
I think it will be better to add a **header** element in the **h1** tag, as it  represents a container for introductory content, and the h1 is the introductory part of the survey form